app/views/users/show.html.erb

```diff
@@ -1,5 +1,10 @@
 <div>
   <h1><%= @user.email %></h2><br>
+    <% if @user.profile_photo.present? %>
+      <img src="<%= @user.profile_photo %>" width="200" />
+    <% else %>
+      <img src="http://sellleadsucceed.files.wordpress.com/2014/02/bigstock-silhouette-with-a-question-mar-59367497.jpg" width="200" />
```

Comment:
You can move this logic to the uploader. Carrierwave provides some functionality for specifying a default image when none are provided: https://github.com/carrierwaveuploader/carrierwave#providing-a-default-url